### ISSUE TITLE
Suggest accessibility: 'read more' not accessible

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#3859]
1. If a suggestion has a 'read more' option this is not announced to the user.
2. The read more action is not accessible via keyboard 

Ping @joaomoreno for 2.
